docs/app/Examples/elements/List/Types/ListListExample.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListListExample extends Component {
+  render() {
+    return (
+      <List>
+        <ListItem description='Apples' />
+        <ListItem description='Pears' />
+        <ListItem description='Oranges' />
+      </List>
+    );
+  }
+}
```

Comment:
Only did one example for the basic list - Semantic has a handful, would adding some variations be useful or necessary?